test/powershell/Modules/Microsoft.PowerShell.Security/certificateCommon.psm1

```diff
@@ -62,7 +62,7 @@ OksttXT1kXf+aez9EzDlsgQU4ck78h0WTy01zHLwSKNWK4wFFQM=
 
     $dataEnciphermentCert = $dataEnciphermentCert -replace '\s',''
     $certBytes = [Convert]::FromBase64String($dataEnciphermentCert)
-    $certLocation = Join-Path $TestDrive "ProtectedEventLogging.pfx"
+    $certLocation = Join-Path ([System.IO.Path]::GetTempPath()) "ProtectedEventLogging.pfx"
```

Comment:
What's the reason for this change and the one below?

Reply:
Was hitting errors in the test relying on this module, but because it's using `$TestDrive`, it was difficult to isolate the problem since you can't import this directly as `$TestDrive` is `$null`.

Reply:
I feel we should continue with `$TestDrive` since it's what Pester encourages, we use it in many other tests and most importantly it gets cleaned up (while I've seen a few issues where `[path]::GetTempPath()` not only isn't, but can't be for some reason).

Usually in these scenarios I use a `Wait-Debugger`, but one alternative would be to set `$TestDrive = [path]::gettemppath()` in the body where it's needed.

Reply:
I'll update it to set $testdrive if it's $null